ai/mcp-server.md

```diff
@@ -16,7 +16,7 @@ The Telerik Blazor [MCP Server](https://modelcontextprotocol.io/introduction) le
 
 To use the Telerik Blazor MCP server, you need:
 
-* A [compatible MCP client (IDE, code editor or app)](https://modelcontextprotocol.io/clients) that supports *MCP tools*.
+* A [compatible MCP client (IDE, code editor or app)](https://modelcontextprotocol.io/clients) that supports *MCP tools*. Using the latest version of the MCP client is recommended.
```

Comment:
The italic formatting here is unnecessary and does not render all that well. 

<img width="648" height="65" alt="image" src="https://github.com/user-attachments/assets/d3c4814f-7284-4899-b724-45e62da8e81c" />